AI-InvestiBot/implementation.py

```diff
@@ -165,16 +177,19 @@ def update_models(models, total_info_keys):
     i = 0
     print(sorted_models)
     for model, profit in sorted_models:
-        if RESOURCE_MANAGER.is_in_portfolio(model[0].stock_symbol) and i<MAX_HOLD_INDEX:
-            RESOURCE_MANAGER.sell()
+        if manager.is_in_portfolio(model[0].stock_symbol) and i<MAX_HOLD_INDEX:
+            manager.sell()
         if profit < RISK_REWARD_RATIO:
             break
-        RESOURCE_MANAGER.buy(model[0].stock_symbol)
+        manager.buy(model[0].stock_symbol)
         i += 1
 
 
-def run_loop(models, total_info_keys) -> None:
+def run_loop(models, total_info_keys, manager: Optional[ResourceManager]=None) -> None:
```

Comment:
i see the while true loop here, but manager needs to be specified,
if not it will be using AWS.

Reply:
?
The while true loop is inside of the function. If the manager is None, the project will set the manager.